test-common/src/main/kotlin/io/opentelemetry/android/test/common/OpenTelemetryTestUtils.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.test.common
+
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.trace.Span
+import io.opentelemetry.sdk.OpenTelemetrySdk
+import io.opentelemetry.sdk.trace.SdkTracerProvider
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor
+import io.opentelemetry.sdk.trace.export.SpanExporter
+
+object OpenTelemetryTestUtils {
+    lateinit var openTelemetry: OpenTelemetry
```

Comment:
Maybe after the okhttp inst changes, I think we should update this to avoid relying on storing the otel instance statically for every test. Seems like a test extension/rule could help.